src/components/gx-query/query-chat/query-chat.tsx

```diff
@@ -58,6 +58,22 @@ export class QueryChat implements GxComponent {
    * In this case the connector must be told the query to execute, either by name (via the objectName property) or giving a full serialized query (via the query property)
    */
   @Prop() readonly metadataName = "";
+  /**
+   * Base URL of the server
+   */
+  @Prop() readonly baseUrl = process.env.GENEXUS_QUERY_URL;
+  /**
+   * Authentication API Key
+   */
+  @Prop() readonly apiKey = process.env.GENEXUS_API_KEY;
+  /**
+   * Authentication Saia Token
+   */
+  @Prop() readonly saiaToken = process.env.GENEXUS_SAIA_TOKEN;
+  /**
+   * Optional Saia user ID
+   */
+  @Prop() readonly saiaUserId = process.env.GENEXUS_SAIA_USER_ID;
```

Comment:
This is an interesting solution, because `process.env` is only defined in the NodeJS runtime, but the component are used in a browser environment. Can we clarify how this works?

Reply:
I thought this better. The environment are unnecessary.
I left these only to run the storybook